docs/project/reward-algorithm V2.mdx

```diff
@@ -1,8 +1,8 @@
 ---
-id: reward-algorithm_v2
-title: Reward Algorithm_v2
-sidebar_label: Reward Algorithm_v2
-slug: /reward-algorithm_v2
+id: reward-algorithm V2
+title: Reward Algorithm V2
+sidebar_label: Reward Algorithm V2
+slug: /reward-algorithm V2

```

Comment:
@ipanagiv Please keep "Reward Algorithm V2" only on `title` and `sidebar_label` and "reward-algorithm-v2" on `id` and `slug`.

It's better not to use spaces on `id` and on the `slug`.

Reply:
Thanks